server.js

```diff
@@ -123,6 +140,32 @@ app.use(bodyParser.urlencoded({
   extended: true
 }))
 
+// Set up v6 app for backwards compatibility
+if (useV6) {
+  var v6Views = [
+    path.join(__dirname, '/node_modules/govuk_template_jinja/views/layouts'),
+    path.join(__dirname, '/app/v6/views/'),
+    path.join(__dirname, '/lib/')
+  ]
+
+  var nunjucksV6Env = nunjucks.configure(v6Views, {
+    autoescape: true,
+    express: v6App,
+    noCache: true,
+    watch: true
+  })
```

Comment:
As these arguments are now set in three different places in the file, I wonder if we might want to have them as a shared variable to reduce duplication. But if the intention is to have the v6 code here for the short term only then the current approach is okay for me 🙂

Reply:
good idea, but the `express` property is different in each one so I think this is tricky